checks/raw/binary_artifact.go

```diff
@@ -103,3 +113,16 @@ var checkBinaryFileContent fileparser.DoWhileTrueOnFileContent = func(path strin
 
 	return true, nil
 }
+
+// TODO: refine this function.
```

Comment:
Nit: make the TODO more actionable. If the refinement refers to using Makefile/YAML parsers, then mention that explicitly here or create an issue and link to it here.